hardhat-example/scripts/contractCall.js

```diff
@@ -0,0 +1,12 @@
+const hre = require('hardhat');
+
+module.exports = async (address, msg) => {
+  const provider = new hre.ethers.providers.JsonRpcProvider('http://localhost:7546');
```

Comment:
Relay endpoint should be an env variable so it's configureable.
Could also make it an optional param which if left out defaults to localhost.
DO this for the other scripts also